spec/deneb/blobsidecar_yaml.go

```diff
@@ -18,33 +18,35 @@ import (
 	"encoding/json"
 	"fmt"
 
+	"github.com/attestantio/go-eth2-client/spec/phase0"
 	"github.com/goccy/go-yaml"
 	"github.com/pkg/errors"
 )
 
 // blobSidecarYAML is the spec representation of the struct.
 type blobSidecarYAML struct {
-	BlockRoot       string `yaml:"block_root"`
-	Index           uint64 `yaml:"index"`
-	Slot            uint64 `yaml:"slot"`
-	BlockParentRoot string `yaml:"block_parent_root"`
-	ProposerIndex   uint64 `yaml:"proposer_index"`
-	Blob            string `yaml:"blob"`
-	KzgCommitment   string `yaml:"kzg_commitment"`
-	KzgProof        string `yaml:"kzg_proof"`
+	Index                       uint64                          `yaml:"index"`
+	Blob                        string                          `yaml:"blob"`
+	KZGCommitment               string                          `yaml:"kzg_commitment"`
+	KZGProof                    string                          `yaml:"kzg_proof"`
+	SignedBlockHeader           *phase0.SignedBeaconBlockHeader `yaml:"signed_block_header"`
+	KZGCommitmentInclusionProof [17]string                      `yaml:"kzg_commitment_inclusion_proof"`
```

Comment:
why not use `kzgCommitmentProofElements`?

Reply:
Good point.  Updated.